USAGE.md

```diff
@@ -12,6 +12,9 @@ Markdown files are identified by their `.md` extension and ordered according to
 For basic formatting, check out the [CommonMark Help](https://commonmark.org/help/) page for an introduction to the formatting options provided by standard markdown.
 In addition, Manubot supports an extended version of markdown, tailored for scholarly writing, which includes [Pandoc's Markdown](https://pandoc.org/MANUAL.html#pandocs-markdown) and the extensions discussed below.
 
+The `content/02.delete-me.md` file in the Rootstock repository shows many of the elements and formatting options supported by Manubot.
+See the [raw markdown](https://gitlab.com/manubot/rootstock/blob/master/content/02.delete-me.md#L) in this file and compare it to the [rendered manuscript](https://manubot.github.io/rootstock/).
```

Comment:
Meant to help address https://github.com/manubot/rootstock/issues/288#issuecomment-567065243.

CC @Nebucatnetzer and @vincerubinetti

Reply:
Using GitLab link to raw markdown as a workaround for https://github.com/isaacs/github/issues/297. Will be slightly confusing, but not sure what is best.